packages/config/src/config/_app.js

```diff
@@ -70,6 +70,8 @@ export default () => ({
     validate: true,
     asyncData: true,
     fetch: true,
+    hookablePlugins: false,
+    parallelPlugins: false,
```

Comment:
I'm **strongly** against introducing these two new configs as global configs. It fragments entire ecosystem! 

Instead, we should add support per/plugin for hook and parallel. A nuxt hook/module can enable it for all but it is then the risk of __consumers__. Not all plugins/modules are designed to work like this and depending on hooks. -- it is a per-plugin feature :)

Reply:
### `parallelPlugins`

- Parallel loading could be a breaking change. Module/plugin authors could make mistakes, this feature could also be used to forcibly run every plugin synchronous in case that happens 
- Consumers are not always in control of plugins, eg when they are added by 3rd party stuff
- Prosumers should be able to override any default behaviour of less-well-maintained 3rd party stuff if they know it should be save
  - This is probably also half a reason for the per-plugin argument, but we have no mechanisms atm in place for prosumers to override module plugin behaviour :)

### `hookablePlugins`
- Thats actually why I mentioned it should probably by default be `true`, we could then also do an extra pass on the used plugins to check if `arguments.length === 3` to enable it on-demand (but that doesnt prevent Hookable from being included in the bundle)

### Both
- You should also see these features as options for prosumers to fully optimize their Nuxt projects like the idea is of the other features. The plugin stuff is also used on the client so we should provide optimization options that just skips all the management logic that is needed for it

Reply:
- In the case of "A superpower user that optimizes its project with full responsibility of enabling flag" she/he can also add a hook to enable this flag(s) for all plugins. including 3rd party ones. At least with a chance to partially do it.

- I think hookable plugins are parallel plugins, however, they are toward a single goal, are completely separate addons to nuxt. discussing in a single PR makes it just confusing.

Reply:
Regarding `arguments.length` check, we can instead optionally export an explicit meta-object from plugins :)

Reply:
> instead optionally export an explicit meta-object from plugins

Hmm, I'm not in favor of having to export meta stuff because _describing_ how something works isnt really _kiss_ and always introduces the chance of the description being wrong. It'd be much better if we could deduct this automatically from syntax

Reply:
Detecting from syntax can be tricky and unstable. Adding separate args for each plugin helper makes it not scalable (like adding more features for plugin export)